targets/XPlatCppSdk/source/cppsdk/source/playfab/PlayFabCallRequestContainer.cpp

```diff
@@ -2,20 +2,41 @@
 
 #include <playfab/PlayFabCallRequestContainer.h>
 
+#ifndef _WIN32
+#include <jsoncpp/json/reader.h>
+#endif
+#ifdef _WIN32
+#include <json/reader.h>
+#endif
+
+
 namespace PlayFab
 {
-    CallRequestContainer::CallRequestContainer() :
+    CallRequestContainer::CallRequestContainer(const CallRequestContainerBase& reqContainer) :
+        CallRequestContainerBase(reqContainer),
         curlHandle(nullptr),
         curlHttpHeaders(nullptr),
-        customData(nullptr),
         finished(false),
         responseString(""),
         responseJson(Json::Value::null),
         errorWrapper(),
-        internalCallback(nullptr),
         successCallback(nullptr),
         errorCallback(nullptr)
     {
+        errorWrapper.UrlPath = reqContainer.getUrl();
+
+        Json::Value request;
+        Json::Reader reader;
+        bool parsingSuccessful = reader.parse(reqContainer.getRequestBody(), request);
+
+        if (parsingSuccessful)
+        {
+            errorWrapper.Request = request;
+        }
+        else
+        {
+            throw new std::invalid_argument("The requestBody string is not in a valid Json format.");
```

Comment:
>throw new std::invalid_argument("The requestBody string is not in a valid Json format."); [](start = 12, length = 89)

This can take out the game - should we just report it ?

Reply:
We will keep it default then. There is nowhere to report.

---
In reply to: [220346639](https://github.com/PlayFab/SDKGenerator/pull/295#discussion_r220346639) [](ancestors = 220346639)